packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -2695,6 +2717,63 @@ export namespace Prisma {
   }
       
 
+  /**
+   * Post findFirstOrThrow
+   */
+  export type PostFindFirstOrThrowArgs = {
```

Comment:
I think these args a really the same as `PostFindFirstArgs`, could we maybe not emit these or is it too difficult?

Reply:
Good call, I did the code changes in https://github.com/prisma/prisma/pull/16162/commits/1b806115f3619bbfc6cedf5e961afe4e704d17ce and pushed the updated snapshots in https://github.com/prisma/prisma/pull/16162/commits/b6decd849ccf0a02778283813ecf10968d661f65